testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/ServiceTest.scala

```diff
@@ -68,6 +68,8 @@ object ServiceTest {
   private val KafkaBrokerModule = "com.lightbend.lagom.internal.javadsl.broker.kafka.KafkaBrokerModule"
   private val KafkaClientModule = "com.lightbend.lagom.javadsl.broker.kafka.KafkaClientModule"
 
+  private val LagomTestConfigResource: String = "lagom-test-embedded-cassandra.yaml"
+
```

Comment:
I chose it for consistency with `CassandraLauncher.DefaultTestConfigResource`.

I guess this is the recommended Scala style for constants: http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

> Constant names should be in upper camel case.

So it breaks with the all-caps convention used by Java, etc.